app/assets/javascripts/components/Newsletter.js

```diff
@@ -0,0 +1,60 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+
+  'use strict';
+
+  var Newsletter,
+      defaultConfig = {};
+
+  Newsletter = function($el, config) {
+    Newsletter.baseConstructor.call(this, $el, config, defaultConfig);
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(Newsletter);
+
+  /**
+  * @param {Promise} initialised
+  */
+  Newsletter.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this.showInArticle();
+    return this;
+  };
+
+  Newsletter.prototype.showInArticle = function() {
+    var $panel = this.getPanel();
+    var $article = $('[data-article]');
```

Comment:
Should probably be `[data-dough-newsletter-article]` to keep it consistent with other Dough components.